aspnetcore/toc.yml

```diff
@@ -1180,6 +1180,36 @@ items:
           - name: Next steps
             displayName: azure, deploy, publish
             href: /dotnet/architecture/devops-for-aspnet-developers/next-steps
+      - name: Kestrel
+        displayName: deploy, publish, server, Kestrel
+        items:
+          - name: Overview
+            displayName: deploy, publish, server
```

Comment:
What does `displayName` do?

Reply:
In the text box at the top of the TOC you can type in a filter expression.  `displayName` lets you specify additional words that will respond to the filter expression. If the item `name` is `Overview` and you don't have `displayName` specified, and you enter "deploy" in the filter box, you won't see this toc item in the filtered results.  If you have `displayName: deploy` you'll get this item in the list when you filter on "deploy".